gtfs/spec/en/reference.md

```diff
@@ -209,244 +211,183 @@ The example below is valid, with distinct blocks every day of the week.
-|   |  | | The units used for **shape_dist_traveled** in the shapes.txt file must match the units that are used for this field in the [stop_times.txt](#stop_timestxt) file. |
-|   |  | | For example, if a bus travels along the three points defined above for A_shp, the additional **shape_dist_traveled** values (shown here in kilometers) would look like this: |
-|   |  | | `A_shp,37.61956,-122.48161,0,0` |
-|   |  | | `A_shp,37.64430,-122.41070,6,6.8310` |
-|   |  | | `A_shp,37.65863,-122.30839,11,15.8765` |
+|  shape_id | ID | **Required** | Identifies a shape. |
+|  shape\_pt\_lat | Latitude | **Required** | Latitude of a shape point. Each record in [shapes.txt](#shapestxt) represents a shape point used to define the shape. |
+|  shape\_pt\_lon | Longitude | **Required** | Longitude of a shape point. |
+|  shape\_pt\_sequence | Non-negative integer | **Required** | Sequence in which the shape points connect to form the shape. Values must increase along the trip but do not need to be consecutive integers. |
+|   |  | | *Ex. If the shape "A_shp" has three points in its definition, the [shapes.txt](#shapestxt) file might contain these records to define the shape:* <br> `shape_id,shape_pt_lat,shape_pt_lon,shape_pt_sequence` <br> `A_shp,37.61956,-122.48161,0` <br> `A_shp,37.64430,-122.41070,6` <br> `A_shp,37.65863,-122.30839,11` |
+|  shape\_dist\_traveled | Non-negative float | Optional | Actual distance traveled along the shape from the first shape point to the point specified in this record. Used by trip planners to show the correct portion of the shape on a map. Values must increase along with `shape_pt_sequence`; they cannot be used to show reverse travel along a route. Distance units must be consistent with those used in [stop_times.txt](#stop_timestxt). |
+|   |  | | *Ex. If a bus travels along the three points defined above for A_shp, the additional `shape_dist_traveled` values (shown here in kilometers) would look like this:* <br> `shape_id,shape_pt_lat,shape_pt_lon,shape_pt_sequence,shape_dist_traveled` <br> `A_shp,37.61956,-122.48161,0,0`<br>`A_shp,37.64430,-122.41070,6,6.8310` <br> `A_shp,37.65863,-122.30839,11,15.8765` |
 
 ### frequencies.txt
 
 File: **Optional**
 
-This table is intended to represent schedules that don't have a fixed list of stop times. When trips are defined in frequencies.txt, the trip planner ignores the absolute values of the **arrival_time** and **departure_time** fields for those trips in [stop_times.txt](#stop_timestxt). Instead, the **stop_times** table defines the sequence of stops and the time difference between each stop.
+[Frequencies.txt](#frequenciestxt) represents scheduled trips that do not have a fixed list of stop times. Arrival and departure times based from information defined in [frequencies.txt](#frequenciestxt), override the the values of associated `stop_times.arrival_time` and `stop_times.departure_time`.
```

Comment:
change to:

"[Frequencies.txt](#frequenciestxt) represents trips that operate on regular headways."

[see discussion about the frequencies.txt file above; justification for this change: if exact_times=0, then trips are NOT "scheduled". if exact_times=1, then there is a fixed list of stop times]

Reply:
Per my earlier comment, I'd suggest change to 

>Frequencies.txt represents trips that operate on regular headways (time between trips) for both frequency-based service (exact_times=0) and a compressed representation of scheduled service (exact_times=1).